frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -78,3 +77,21 @@ export const UpsellCardComponent = styled.div<Variants>`
   border: 1px solid ${upsellColors.secondary};
   background-color: var(--mb-color-bg-white);
 `;
+
+export const UpsellBannerComponent = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+  padding: 1rem;
+  overflow: hidden;
+  border: 1px solid var(--mb-color-border);
+  border-radius: 0.5rem;
+`;
```

Comment:
we should not be making new styled components. let's start a CSS modules file for upsells.

Reply:
I created new CSS modules file, but avoided moving ALL upsells to it for now (to reduce the diff).
Tracking it here [ENG-14650](https://linear.app/metabase-inc/issue/ENG-14650/convert-all-upsells-from-styled-components-to-css-modules)